libr/core/linux_heap_glibc.c

```diff
@@ -1284,6 +1311,12 @@ static int GH(cmd_dbg_map_heap_glibc)(RCore *core, const char *input) {
 		return false;
 	}
 
+	if (GH(is_tcache) (core)){
```

Comment:
just use GH(is_tcache)(core)) as argument to RConfig.set_i().

also, the argument should be numeric, not a string, otherwise its always true